hushline/routes.py

```diff
@@ -129,7 +132,7 @@ def submit_message(username: str) -> Response | str:
                 email_content = encrypted_content if encrypted_content else content
                 if not encrypted_content:
                     flash("⛔️ Failed to encrypt message with PGP key.", "error")
-                    return redirect(url_for("submit_message", username=username))
+                    return redirect(url_for("submit_message", username=username)), 400
```

Comment:
cannot redirect with 400

Reply:
👍

Reply:
there's still 4xx status codes for redirects in the PR

Reply:
👍